docs/index.md

```diff
@@ -1 +1,123 @@
+# Introduction
 
+Lightstone Platform Product is a library intended to support product developers to surface their products through and leverage the Lightstone platform.
+
+The Lightstone Platform comprises of a product flow that always follows the sames steps to surface a product through the product flow web component. This library serves as the basis for all products that are built and surfaced through the Lightstone Platform, and contains all needed models and end points to be able to successfully surface your product.
+# Product Implementation
+## Product Flow Explained
+
+When a tenant on the platform initiates a new product flow, the flow goes through various steps to determine what should be displayed to the user. The flow is made up of the following steps:
+
+- Initiation
+    - The tenant is responsible for initiating the start request to platform, this request is used by all tenants on platform to be able to surface a product through platform.
+- Legal
+    - Platform gets the data from the product catalogue for the product that was specified when initiating the flow and determines if there is any legal requirements for this flow to proceed and will automatically get any inputs from the user if required.
+- Start
+    - This is the first time that platform will need to communicate with your new product, platform will do an API request to your product based on the configured base url for your product by calling the Start end-point.  Your product will return a relevant response as per the steps below.
+- Verify Inputs
+    - If the response in the start end-point indicated that inputs was required from the user, platform will surface your web component to the user, it will then collect that data from the user and send that data back to your product using the Verify Inputs end-point, your product will then determine if the input received is valid and if the flow can proceed
```

Comment:
```suggestion
    - If the response in the start end-point indicated that inputs were required from the user, the platform will surface your web component to the user. It will then collect that data from the user and send that data back to your product using the Verify Inputs end-point. Your product will then determine if the input received is valid and if the flow can proceed
```